docs/src/pages/components/menus/ContextMenu.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import Button from '@material-ui/core/Button';
+
+const ContextMenu = () => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [mouseX, setMouseX] = React.useState(null);
+  const [mouseY, setMouseY] = React.useState(null);
+
+  const handleClick = event => {
+    event.preventDefault();
+    setMouseY(event.clientY - 4);
+    setMouseX(event.clientX + 2);
+    setAnchorEl(event.currentTarget);
+  };
+
+  useEffect(() => {}, [mouseX, mouseY]);
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div
+      className="context-wrapper"
+      aria-controls="simple-menu"
+      aria-haspopup="true"
+      onContextMenu={handleClick}
+    >
+      <Button>Right Click To Open Menu</Button>
```

Comment:
I would propose using a similar content to https://material-ui.com/components/popper/#faked-reference-object instead of a button.

- https://ej2.syncfusion.com/javascript/documentation/context-menu/es5-getting-started/
- https://www.telerik.com/kendo-react-ui/components/layout/menu/context-menu/